ui/src/constants.js

```diff
@@ -16,6 +16,12 @@ export const STATUS_READY = 'ready';
 export const STATUS_NOT_READY = 'not_ready';
 export const STATUS_UNKNOWN = 'unknown';
 export const STATUS_BOUND = 'Bound';
+export const STATUS_PENDING = 'Pending';
+export const STATUS_TERMINATING = 'Terminating';
+export const STATUS_FAILED = 'Failed';
+export const STATUS_AVAILABLE = 'Available';
+export const STATUS_RELEASED = 'Released';
+export const STATUS_VOLUME_UNKNOWN = 'Unknown';
```

Comment:
We changed the `STATUS_UNKNOWN` to `STATUS_API_UNKNOWN` because the lowercase `unknow` is come from the API.  So later we can reuse STATUS_UNKNOWN in any other object.

Reply:
Works for me :slightly_smiling_face: